README.md

```diff
@@ -28,25 +27,26 @@ If you can't use our guide, please ask about what you don't understand or consid
 Note that this implementation of Electrum server is optimized for **personal/small-scale (family/friends) usage**.
 It's a bad idea to run it publicly as it'd expose you to DoS and maybe also other attacks.
 If you want to run a public server you may be interested in the [Blockstream fork of electrs](https://github.com/Blockstream/electrs)
-which is better optimized for public usage at the cost of consuming *significantly* more resources.
+which is better optimized for public usage at the cost of consuming _significantly_ more resources.
```

Comment:
Are those changes required?

Reply:
Oh, no, that's just my editor's auto-format plugin changed it while save the file. I'll remove this change, soon.